lib/pronto/config_file.rb

```diff
@@ -35,8 +36,10 @@ class ConfigFile
       'format' => DEFAULT_MESSAGE_FORMAT
     }.freeze
 
-    def initialize(path = '.pronto.yml')
-      @path = path
+    def initialize
+      @path = ENV.fetch('PRONTO_CONFIG', DEFAULT_FILE_PATH)
+      return if @path == DEFAULT_FILE_PATH || File.exist?(@path)
```

Comment:
`ENV['PRONTO_CONFIG']` will always be set because it's the default value of the `--config` option in `Pronto::CLI`.

An option could be to drop the default value for the `--config` option and keep the same logic in this code to fall back to `.pronto.yml` (like you're suggesting).